### PR TITLE
Add Colab and Python version

### LINKED ISSUE #693: BUG-Document installation on Colab
**Bug description**

[This issue](https://github.com/Avaiga/taipy/issues/365) mentions that Taipy 3.0 cannot be installed on Colab, and the Taipy doc doesn't help.

**Expected change**

The documentation must explicitly state the workaround on the installation page, maybe in the related Tip section.

### LINKED ISSUE #731: BUG- Wrong pip Symbol used in Documentation
***Description***
There is a bug which i found while going through the documentation of taipy on the official website. The documentation shows that for Installation with pip (https://docs.taipy.io/en/latest/getting_started/#installation-with-pip)
Prerequisites: Ensure you have Python (version 3.8 or higher) and [pip](https://pip.pypa.io/)installed.
Installation Command: Run the following in your terminal or command prompt: $ pip install taipy.

*How to reproduce*
$ pip install taipy.

*Expected behavior*
Here it is shown that we have to use the $ pip, but when i performed it on many platforms and ide such as collab , juypter notebook , vs code and as taipy being a tool for data science we uses the above notebook only for collaboration, the code was showing the error of SyntaxError: invalid syntax.

In Order to atleast initiate the installation we have to use ! syntax instead of $ in the code. After using the ! Syntax we get the error of blinker. Which has to be solved separately in order to completely install taipy.

Screenshot 2024-01-20 231913
Screenshot 2024-01-20 232008

Runtime Environment
Software Used - Google Colab , Juypter notebook

Os- Windows 11
Browser - Chrome

### LINKED ISSUE #734: Error-cannot uninstall blinker at the time of pip install taipy
**Description**
The time we use command **`!pip install taipy`**, after all the processing of necessary packages at last we get an error stating that:

**ERROR: Cannot uninstall 'blinker'. It is a distutils installed project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall.**

**Acceptance Criteria**
To correct this particular issue, after hitting and trial I came up with the thing of using the next command as:

**`!pip install --ignore-installed taipy`**

After running this code I got the following message and after that I got to Restart session 

**WARNING: Upgrading ipython, ipykernel, tornado, prompt-toolkit or pyzmq can cause your runtime to repeatedly crash or behave in unexpected ways and is not recommended. If your runtime won't connect or execute code, you can reset it with "Disconnect and delete runtime" from the "Runtime" menu.
WARNING: The following packages were previously imported in this runtime:
  [_distutils_hack,certifi,dateutil,numpy,pkg_resources,psutil,setuptools,six,tornado]
You must restart the runtime in order to use newly installed versions.**

### Issue in pip install taipy 
pip install taipy

### Correct format of pip install taipy
correct pip install taipy
correct pip install taipy 2

### PR DESCRIPTION
**Pull request by FlorianJacta**
Maybe related to: https://github.com/Avaiga/taipy/issues/730

Resolves: https://github.com/Avaiga/taipy/issues/731

Resolves: https://github.com/Avaiga/taipy/issues/734

Resolves: https://github.com/Avaiga/taipy-doc/issues/693

image

### CODE REVIEW

File: docs/getting_started/index.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ first application with ease.
 
 # Installation with pip
 
-1. **Prerequisites**: Ensure you have Python (version 3.8 or higher) and 
+1. **Prerequisites**: Ensure you have Python (**version between 3.8 and 3.11**) and
```
Comment (FabienLelaquais) [inline docs/getting_started/index.md:15]: We actually plan to support 3.12 in 3.1.

File: docs/installation/index.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ powerful web applications.
 
 # Prerequisite
 
-Before installing Taipy, ensure you have Python (version 3.8 or above) and 
+Before installing Taipy, ensure you have Python (**between version 3.8 and 3.11**) and
```
Comment (FabienLelaquais) [inline docs/installation/index.md:13]: Probably not

File: docs/getting_started/index.md [hunk 3]
```diff
@@ -12,12 +12,12 @@ first application with ease.
 
 # Installation with pip
 
-1. **Prerequisites**: Ensure you have Python (version 3.8 or higher) and 
+1. **Prerequisites**: Ensure you have Python (**version between 3.8 and 3.11**) and 
     [pip](https://pip.pypa.io)installed.
 
 2. **Installation Command**: Run the following in your terminal or command prompt:
     ``` console
```
Comment (FabienLelaquais) [inline docs/getting_started/index.md:19]: No space after the backticks

File: docs/installation/index.md [hunk 4]
```diff
@@ -44,6 +44,33 @@ $ pip install taipy
...
+and runs entirely in the cloud. To install Taipy in Google Colab, follow these simple 
+steps:
+
+1. **Open a new Colab notebook**: Visit [Google Colab](https://colab.research.google.com) 
+and start a new notebook.
+
+2. **Run the installation command**: In a new cell, enter the following command and run 
+the cell:
+
+    ``` python
```
Comment (FabienLelaquais) [inline docs/installation/index.md]:
```suggestion
    ```python
```

File: docs/installation/index.md [hunk 5]
```diff
@@ -44,6 +44,33 @@ $ pip install taipy
...
+    !pip install --ignore-installed taipy
+    ```
+
+    This command installs the latest stable release of Taipy in your Colab environment.
+
+3. **Start building your app**: Follow this 
+[tip](../knowledge_base/tips/colab_with_ngrok/index.md) to build and your Taipy web
```
Comment (FabienLelaquais) [inline docs/installation/index.md]: "to build and your Taipy"?
I don't understand the sentence

**Review by FabienLelaquais — APPROVED** [review]
Minor details.

File: docs/getting_started/index.md (hunk 1 above)
Comment (FlorianJacta) [reply docs/getting_started/index.md:15]: Yes, I know about this. We should rechange it when it is out

File: docs/installation/index.md (hunk 2 above)
Comment (FlorianJacta) [reply docs/installation/index.md:13]: ?

File: docs/getting_started/index.md (hunk 3 above)
Comment (FlorianJacta) [reply docs/getting_started/index.md:19]: I followed the syntax of this page. I will change it for the others also

**Pull request merged**